BigQuery/src/BigQueryClient.php

```diff
@@ -682,4 +682,22 @@ public function timestamp(\DateTimeInterface $value)
     {
         return new Timestamp($value);
     }
+
+    /**
+     * Get a service account for the KMS integration.
+     *
+     * Example:
+     * ```
+     * $serviceAccount = $bigQuery->getServiceAccount();
+     * ```
+     * @return string
+     */
+    public function getServiceAccount()
```

Comment:
We should allow `$options` to be passed in to any requests which trigger network requests, it is how you can set items like timeouts.